USERS.md

```diff
@@ -17,6 +17,7 @@ Currently, the following organizations are **officially** using Argo Workflows:
 1. [Ant Group](https://www.antgroup.com/)
 1. [AppDirect](https://www.appdirect.com/)
 1. [BasisAI](https://basis-ai.com/)
+1. [BEI.RE](https://www.bei.re/)
```

Comment:
how can we verify your relationship with this org please?

Reply:
<img width="1164" alt="Screen Shot 2020-11-24 at 12 36 43 AM" src="https://user-images.githubusercontent.com/44732600/100069212-50bd1480-2ded-11eb-954f-38170316af75.png">